package.json

```diff
@@ -46,10 +46,11 @@
     "release": "np --no-release-draft",
     "pretest": "npm run lint",
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
-    "test-coverage": "npm run test --ignore-script -- --coverage",
+    "test:only": "npm test --ignore-scripts",
```

Comment:
notice the `s`

Reply:
This change makes `npm run watch` not work: `jest.js` doesn't receive `--watch`.

```console
$ npm run watch -- lib/rules/selector-pseudo-class-no-unknown

> stylelint@15.9.0 watch
> npm run test:only -- --watch lib/rules/selector-pseudo-class-no-unknown


> stylelint@15.9.0 test:only
> npm test --ignore-scripts --watch lib/rules/selector-pseudo-class-no-unknown


> stylelint@15.9.0 test
> node --experimental-vm-modules node_modules/jest/bin/jest.js lib/rules/selector-pseudo-class-no-unknown
```

So, could you revert this change for `test:only` in `package.json`?

In addition, `test-coverage` instead of `test:coverage` is intentional. It is never combined like `npm-run-all lint:*`.

Reply:
It will have to be fixed but in a separate PR.
AFAIK `--ignore-script` doesn't exist.

Reply:
See https://docs.npmjs.com/cli/v9/commands/npm-test#ignore-scripts

Reply:
Iv read it twice I see `--ignore-scripts` not `--ignore-script`.

Reply:
Oh, I missed it, sorry. 🙇🏼 
I'll open PR soon.

Reply:
Opened #6967.

Reply:
If so please add a script to run `test` without `pretest`.

Reply:
Make sense. Opened #6969.